src/assets/js/dynamic-addon-cards.js

```diff
@@ -80,10 +86,21 @@
             }
           }
 
-          // TODO: if `clientApp` is `android`, we should also check the
-          // recommended prop and see whether the add-on can be installed on
-          // Fenix.  We probably don't need to care about Fennec anymore.
-          // Then, we will need to check the min version since the max version
+          if (!disableButton && clientApp === 'android') {
```

Comment:
Should we be using constants for things like `'android'` and `'recommended'`?

Reply:
I think it's fine for now, we don't use them too many times.